rmgpy/cantherm/input.py

```diff
@@ -165,6 +165,10 @@ def species(label, *args, **kwargs):
         job = StatMechJob(species=spec, path=path)
         logging.debug('Added species {0} to a stat mech job.'.format(label))
         jobList.append(job)
+    elif len(args) > 1:
```

Comment:
If you see how the header is written, `args` excludes the species label which is always required.
The code you showed will still run properly since there is only one additional non-keyword argument.

I am clarifying the error message so this is not confusing to the user.